### ISSUE TITLE
azerty keyboard

### ISSUE THREAD

**Issue by parazyt75** [foxlet/bootra1n#56]
Hi, 

Is there a way to change keyboard layout from qwerty to azerty ?

Big thanks

**Comment by dcvwitty21** [comment]
You can try to search it on google on how to change it.

**Comment by parazyt75** [comment]
@dcvwitty21 
What a useful comment !!! thanks a lot for your help, if you don't need anything i will be happy to help you !!!

**Comment by dcvwitty21** [comment]
Yeah i need help.. I have a problem on why my disabled iphone X cant be jailbroken. I tried to jailbreak it using checkra1n 0.10.1 it just stuck on booting... I then my phone will go to the disabled screen
